src/antelope/stores/utils/contracts/EvmContract.ts

```diff
@@ -138,8 +138,13 @@ export default class EvmContract {
             throw new AntelopeError('antelope.utils.error_contract_instance');
         }
         const signer = await this._manager?.getSigner();
+        let provider;
 
-        return new ethers.Contract(this.address, this.abi, signer);
+        if (!signer) {
+            provider = await this._manager?.getWeb3Provider();
+        }
```

Comment:
addresses the issue where a logged-out user cannot view NFTs because there is no signer when logged out